akka-http-core/src/main/scala/akka/http/impl/engine/client/PoolFlow.scala

```diff
@@ -72,7 +72,7 @@ private object PoolFlow {
     connectionFlow: Flow[HttpRequest, HttpResponse, Future[Http.OutgoingConnection]],
     settings:       ConnectionPoolSettings, log: LoggingAdapter)(
     implicit
-    system: ActorSystem, fm: Materializer): Flow[RequestContext, ResponseContext, NotUsed] =
+    system: ActorSystem, mat: Materializer): Flow[RequestContext, ResponseContext, NotUsed] =
```

Comment:
heh, this reminds me of the time where we had different materializers for graphs and flows... I forgot completely about that :)